vignettes/sjtbasic.Rmd

```diff
@@ -151,10 +139,10 @@ sjt.frq(efc$e42dep,
                    css.lasttablerow = 'border-top: 1px solid; border-bottom: none;', 
                    css.summary = 'color: blue;'))
 ```
-`r sjt.frq(efc$e42dep, no.output=TRUE, CSS=list(css.centeralign='text-align: left;', css.caption='font-weight: normal; font-style: italic;', css.firsttablecol='font-weight: bold;', css.lasttablerow='border-top: 1px solid; border-bottom: none;', css.summary='color: blue;'))$knitr`
```

Comment:
By removing this, `eval = FALSE` in line 134 must be removed. Same for all eval=FALSE in following code-chunks, since these code-chunks can now be evaluated.